.gitlab-ci.yml

```diff
@@ -21,12 +20,12 @@ include:
 
 test:unit:
   stage: test
-  image: golang:1.17
+  image: golang:1.21
```

Comment:
Same here, we need to keep compatibility for 1.17. Reasoning in [this commit](https://github.com/mendersoftware/mender/commit/c1771542e9e2da4944211c9ccf0201d95c2a52d5).